reorder_python_imports.py

```diff
@@ -651,7 +672,10 @@ def _is_py3(args):  # type: (argparse.Namespace) -> bool
 
 def _six_replaces(args):  # type: (argparse.Namespace) -> List[ImportToReplace]
```

Comment:
this function's name doesn't make sense now

Reply:
fixed